cmd/trivy.go

```diff
@@ -0,0 +1,58 @@
+package cmd
+
+import (
+	"dynamic-buildkite-template/generator"
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+var (
+	trivyPluginConfig generator.TrivyPluginConfig
+)
+
+var trivyCmd = &cobra.Command{
+	Use:   "trivy",
+	Short: "Generates trivy plugin step for the given configurations",
+	Long: `
+Generates trivy plugin step for the given configurations
+	`,
+	Run: func(cmd *cobra.Command, args []string) {
+		CreateGenerator(cmd, args)
+	},
+}
+
+func init() {
+
+	trivyCmd.Flags().Int("exit-code", 0, "Controls whether the security scan is blocking or not for trivy buildkite plugin")
+	trivyCmd.Flags().String("timeout", "5m0s", "Controls the maximum amount of time a scan will run for trivy buildkite plugin")
+	trivyCmd.Flags().String("severity", "HIGH,CRITICAL", "Controls the severity of the vulnerabilities to be scanned for trivy buildkite plugin")
+	trivyCmd.Flags().Bool("ignore-unfixed", true, "Controls whether to display only fixed vulnerabilities for trivy buildkite plugin")
+	trivyCmd.Flags().String("security-checks", "vuln,config", "Controls the security checks to be performed for trivy buildkite plugin")
+	trivyCmd.Flags().String("skip-files", "", "Controls the files to be skipped during the scan for trivy buildkite plugin")
+	trivyCmd.Flags().String("skip-dirs", "", "Controls the directories to be skipped during the scan for trivy buildkite plugin")
+	trivyCmd.Flags().String("image-ref", "", "Controls the image reference to be scanned for trivy buildkite plugin")
+	trivyCmd.Flags().String("version", "v1.18.2", "Controls the version of trivy to be used for trivy buildkite plugin")
```

Comment:
I still have concerns over hard coding this version but let’s merge this and go from there

Reply:
Okay, Thank you @sfunkhouser I'll take note of it and make the necessary changes.